src/avs/charts/TVLTabLineChart.jsx

```diff
@@ -0,0 +1,322 @@
+            TVL over time
+          </div>
+          <div className="flex gap-x-2 text-foreground-2 text-xs">
+            <span>{getLatestTotals}</span>
+            <span
+              className={`${growthPercentage >= 0 ? 'text-success' : 'text-danger'}`}
+            >
+              {growthPercentage > 0 && '+'}
+              {formatNumber(growthPercentage, false)}%
+            </span>
+          </div>
+        </div>
+
+        <div className="flex flex-1 w-full flex-col gap-y-2 justify-between sm:flex-row sm:justify-end gap-x-2">
+          <Tabs
+            classNames={tabs}
+            defaultSelectedKey="usd"
+            onSelectionChange={handleRateSelectionChange}
+            size="sm"
+          >
+            <Tab key="usd" title="USD" />
+            <Tab key="eth" title="ETH" />
+          </Tabs>
+          <Tabs
+            classNames={tabs}
+            defaultSelectedKey="all"
+            disabledKeys={disabledKeys}
+            onSelectionChange={handleTimelineSelectionChange}
+            size="sm"
+          >
+            <Tab key="1w" title="1W" />
+            <Tab key="1m" title="1M" />
+            <Tab key="3m" title="3M" />
+            <Tab key="1y" title="1Y" />
+            <Tab key="all" title="All" />
+          </Tabs>
+        </div>
+      </div>
+
+      <svg
+        className="touch-pan-y w-full"
+        ref={containerRef}
+        width={width}
+        height={height}
+      >
+        <Group left={margin.left} top={margin.top}>
+          <GridRows
+            scale={scaleValue}
+            width={state.maxX - margin.right}
+            height={state.maxY}
+            numTicks={4}
+            className="[&_line]:stroke-foreground-2 opacity-25"
+          />
+          <GridColumns
+            scale={scaleDate}
+            width={state.maxX}
+            height={state.maxY}
+            numTicks={Math.floor(state.maxX / 120)}
+            className="[&_line]:stroke-foreground-2 opacity-25"
+          />
+          <AxisRight
+            left={state.maxX - margin.right}
+            scale={scaleValue}
+            numTicks={4}
+            tickFormat={v => formatNumber(v, true)}
+            axisLineClassName="stroke-foreground-2"
+            tickClassName="[&_line]:stroke-foreground-2"
+            tickLabelProps={{
+              className: 'text-xs',
+              fill: 'hsl(var(--app-foreground))'
+            }}
+          />
+          <AxisBottom
+            top={state.maxY}
+            scale={scaleDate}
+            numTicks={Math.floor(state.maxX / 120)}
+            tickFormat={formatDate}
+            axisLineClassName="stroke-foreground-2"
+            tickClassName="[&_line]:stroke-foreground-2"
+            tickLabelProps={{
+              className: 'text-xs',
+              fill: 'hsl(var(--app-foreground))'
+            }}
+          />
+          <LinePath
+            className="stroke-dark-blue"
```

Comment:
Please put comment here to replace the hardcoded color. Also, let's make the line smoother.
```suggestion
            className="stroke-dark-blue"
            curve={curveMonotoneX}
```

Don't forget to import `curveMonotoneX`.

Reply:
wdym by hardcoded colour? It should be a prop or a tailwind variable?

Reply:
The class name already says it's blue. Although we can have a green color under the name "blue", it's weird and confusing. There should be a color-invariant name, like we have for fonts -- "foreground-2".